### PR TITLE
Feature/5945 rat profile/6515 coordinator view controller dummies

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
Created the flow to create the AntigenTestProfile
* show information screen on first open
* show create profile is no profile was found
* show profile if profile was created or found

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6515

## Notice
There are some comments left in the code to inform about required changes if the screen gets done by someone else

### CODE REVIEW

**Review by freshking — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileViewController.swift [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		}
+	}
+
+	// MARK: - Protocol DismissHandling
+
+	func wasAttemptedToBeDismissed() {
+		dismiss()
+	}
+
+	// MARK: - Public
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileViewController.swift:65]: Can be removed?

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileViewController.swift [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+	// MARK: - Internal
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileViewController.swift:67]: same

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+	// MARK: - Init
+	init(
+		store: AntigenTestProfileStoring
+	) {
+		self.store = store
+		self.antigenTestProfile = store.antigenTestProfile
+	}
+
+	// MARK: - Overrides
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileViewModel.swift:17]: same here and all below

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfileInformation/AntigenTestProfileInformationViewController.swift [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+		parent?.navigationItem.rightBarButtonItem = dismissHandlingCloseBarButton
+		viewModel.markScreenSeen()
+	}
+
+	// MARK: - Protocol FooterViewHandling
+
+	func didTapFooterViewButton(_ type: FooterViewModel.ButtonType) {
+
+		// this one can get used if data privacy will be removed from footer's secondary button
+//		guard case .primary = type else {
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfileInformation/AntigenTestProfileInformationViewController.swift:44]: can be removed?

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfileInformation/AntigenTestProfileInformationViewController.swift [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+	// MARK: - Protocol DismissHandling
+
+	func wasAttemptedToBeDismissed() {
+		dismiss()
+	}
+
+	// MARK: - Public
+
+	// MARK: - Internal
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfileInformation/AntigenTestProfileInformationViewController.swift:63]: .

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**